package.json

```diff
@@ -10,17 +10,12 @@
         "prod": "NODE_ENV=production",
         "build": "yarn dev webpack",
         "watch": "yarn dev webpack --watch",
-        "test": "yarn dev karma start",
-        "test:watch": "yarn dev karma start"
```

Comment:
you are 
1. removing karma from scripts ("test") - good
2. removing karma from dev dependencies - good
3. adding jest to scripts - good
4. should add jest to dev dependencies to install - now this step is missed, please add jest running "yarn add --dev jest"